### ISSUE TITLE
The element XCSwiftPackageProductDependency is not supported.

### ISSUE THREAD

**Issue by 36v** [peripheryapp/periphery#59]
When I try to run periphery on my Xcode 11.3 Swift project, it aborts quickly with this error message.
```
jakob@mkwnh Pix % periphery scan
Welcome to Periphery!
This guided setup will help you select the appropriate configuration for your project.

* Inspecting project configuration...

error: The element XCSwiftPackageProductDependency is not supported.
```

**Comment by txaiwieser** [comment]
I'm also facing this problem

**Comment by tychop** [comment]
Yep, it makes the whole periphery application worthless...

**Comment by joustava** [comment]
I have a similar issue:
```
error: The element XCRemoteSwiftPackageReference is not supported.
```

**Comment by rcaraway** [comment]
Facing this same issue.

**Comment by ileitch** [comment]
Please see here for the status of Periphery: https://github.com/peripheryapp/periphery/issues/66

**Comment by ileitch** [comment]
This should be fixed in 1.6.0.
